README.md

```diff
@@ -908,12 +908,14 @@ Create a network
 Usage: `nerdctl network create [OPTIONS] NETWORK`
 
 Flags:
+- :whale: `-d, --driver`: Driver to manage the Network
+- :whale: `-o, --opt`: Set driver specific options
```

Comment:
Could you explain the list of the drivers and the opts in this documentation?

Reply:
Done.